src/components/Modals/Send/views/Address.tsx

```diff
@@ -39,6 +45,9 @@ export const Address = () => {
   if (!(asset?.chain && asset?.name)) return null
 
   const adapter = chainAdapters.byChain(asset.chain)
+  const isEthereumChainAdapter = (
```

Comment:
@cjthompson this is fine for now but all the more reason to get the types discriminating properly and have `chainAdapters.byChain` return the properly narrowed type

Reply:
If you agree, I'd like to tackle it on a follow-up PR, spend quite some time struggling with trying to narrow the type and I'd like to focus on it while not working on business logic

Reply:
https://github.com/shapeshift/lib/issues/321